apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -173,16 +173,14 @@ const ReviewApplication = ({ issue, requestIndex }) => {
               mode="negative"
               onClick={onReject}
               icon={<IconCross />}
-            >
-                Reject
-            </ReviewButton>
+              label="Reject"
+            />
             <ReviewButton
-              icon={<IconCheck />}
               mode="positive"
               onClick={onAccept}
-            >
-                Accept
-            </ReviewButton>
+              icon={<IconCheck />}
```

Comment:
Why re-order these props? Looks like they were in alphabetical order before. What order are they in now?